consensus/qbft/src/integration-test/java/org/hyperledger/besu/consensus/qbft/test/FutureRoundTest.java

```diff
@@ -68,11 +68,11 @@ public void messagesForFutureRoundAreNotActionedUntilRoundIsActive() {
     final ConsensusRoundIdentifier subsequentRoundId = new ConsensusRoundIdentifier(1, 6);
     final RoundSpecificPeers subsequentRoles = context.roundSpecificPeers(subsequentRoundId);
 
-    // required remotely received Prepares = quorum-2
+    // required remotely received Prepares = quorum-1
```

Comment:
can we just ditch these comments all together? Given the quorums are the same

Reply:
done